README.md

```diff
@@ -133,15 +141,17 @@ import { IKImage, IKContext, IKUpload } from 'imagekitio-react'
 
 ## Components
 
-The library includes 3 Components:
+The library includes 5 Components:
 
 * [`IKContext`](#IKContext) for defining options like `urlEndpoint`, `publicKey` or `authenticationEndpoint` to all children elements. This component does not render anything.
 * `IKImage` for [image resizing](#image-resizing). This renders a `<img>` tag.
+* `IKVideo` for [video resizing](#video-resizing). This renders a `<video>` tag.
 * `IKUpload`for client-side [file uploading](#file-upload). This renders a `<input type="file">` tag.
+* `IKCore` for [Core SDK](#ikcore), This exposes methods from [ImageKit javascript SDK](https://github.com/imagekit-developer/imagekit-javascript) like url and upload.
 
 ## IKContext
 
-In order to use the SDK, you need to provide it with a few configuration parameters. You can use a parent `IKContext` component to define common options for all children `IKImage` or `IKupload` compoents. For example:
```

Comment:
[Demo application](https://github.com/imagekit-samples/quickstart/tree/master/react) and [docs PR](https://docs.imagekit.io/getting-started/quickstart-guides/react) is missing. Videos and new upload props needs to be integrated there.